src/components/Common/CaseStudyCards/CaseStudyPreview.js

```diff
@@ -59,4 +61,17 @@ const CaseStudyPreview = ({ isTop, caseStudy }) => {
   )
 }
 
+CaseStudyPreview.propTypes = {
+  isTop: PropTypes.bool,
+  caseStudy: PropTypes.shape({
+    title: PropTypes.string.isRequired,
+    slug: PropTypes.string.isRequired,
+    posterImage: PropTypes.object,
+    introSentence: or([
+      PropTypes.string.isRequired,
+      PropTypes.objectOf(PropTypes.string.isRequired).isRequired
```

Comment:
should this not be `shape`? this just means an object with values of a string.

`PropTypes.shape({introSentence: PropTypes.string.isRequired})`